app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :post, class_name: 'Post', foreign_key: 'post_id'
+
+  def update_post_comment_counter
+    post.increment!(:CommentsCounter)
+  end
```

Comment:
- I suggest making your update method across your models private and using rails active callbacks to trigger it